packages/office-ui-fabric-react/src/components/Icon/Icon.types.ts

```diff
@@ -70,16 +65,27 @@ export interface IIconProps extends IBaseProps, React.HTMLAttributes<HTMLElement
    * Gets the styles for an Icon.
    */
   getStyles?: IStyleFunction<IIconStyleProps, IIconStyles>;
+
+  /**
+   * Deprecated: use getStyles.
+   * @deprecated
+   */
+  styles?: IIconStyles;
 }
 
 export interface IIconStyleProps {
   className?: string;
   iconClassName?: string;
   isPlaceholder: boolean;
   isImage: boolean;
+  styles?: Partial<IIconStyles>;
 }
 
 export interface IIconStyles {
-  root: IStyle;
+  root?: IStyle;
+
+  /**
+   * Deprecated. Use 'root'.
```

Comment:
Missing `@deprecated`. Hm would it be helpful to continue tracking the major version these vars were deprecated, to help in deprecated cleanup later?